.prettierrc.json

```diff
@@ -7,5 +7,12 @@
     "trailingComma": "none",
     "bracketSpacing": true,
     "arrowParens": "avoid",
-    "parser": "typescript"
```

Comment:
[From Prettier’s docs](https://prettier.io/docs/en/options.html#parser):

> Prettier automatically infers the parser from the input file path, so you shouldn't have to change this setting.

Therefore, this option can be safely removed.